setup.cfg

```diff
@@ -1,28 +1,79 @@
+[metadata]
+name = eazy
+author = G. Brammer
+author_email = gbrammer@gmail.com
+description = Pythonic photo-zs
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/gbrammer/eazy-py
+license = MIT
+license_files = LICENSE.txt
+project_urls =
+    Documentation = https://eazy-py.readthedocs.io/
+    Source = https://github.com/gbrammer/eazy-py
+    Tracker = https://github.com/gbrammer/eazy-py/issues
+classifiers=
+    Development Status :: 3 - Alpha
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering :: Astronomy
+
+[options]
+python_requires = >=3.7
+install_requires =
+    numpy
+    scipy
+    matplotlib
+    astropy
+    peakutils
+    tqdm
+    h5py
+    astro-sedpy
+    astro-prospector
+    dust_extinction @ git+https://github.com/gbrammer/dust_extinction.git
+    dust_attenuation @ git+https://github.com/gbrammer/dust_attenuation.git
```

Comment:
This will have to point to released packages on pypi in order for this package to be able to be released on pypi, fyi.

Reply:
Thanks @jdavies-st .  In fact it looks like I had actually sent my changes in PRs to the dust modules some time ago, so indeed these can now be the standard `pip` requirements.